jenkins/node-image/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file creates a build environment for building and running kubernetes
+# unit and integration tests
+
+FROM gcr.io/k8s-testimages/kubekins-test:1.6-v20161205-ad918bc
+LABEL authors="Sen Lu <senlu@google.com>"
+
+# Add necessary scripts
+ADD ["node-runner.sh", \
+    "/workspace/"]
+RUN ["chmod", "+x", "/workspace/node-runner.sh"]
+
+# Install gcloud
+RUN curl -fsSL --retry 3 --keepalive-time 2 -o /workspace/google-cloud-sdk.tar.gz 'https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz' \
+    && tar xzf /workspace/google-cloud-sdk.tar.gz -C /workspace \
+    && export CLOUDSDK_CORE_DISABLE_PROMPTS=1 \
+    && /workspace/google-cloud-sdk/install.sh --disable-installation-options --bash-completion=false --path-update=false --usage-reporting=false \
+    && export PATH=/workspace/google-cloud-sdk/bin:${PATH} \
+    && gcloud components install alpha \
+    && gcloud components install beta \
+    && gcloud info
```

Comment:
we don't need `gcloud info` here. this is only useful in test runs, not in container creation.